Cookbook/Technical-Documents/FastlaneMatch.md

```diff
@@ -3,39 +3,63 @@ Fastlane Match
 
 Fastlane match is a tool for managing certificates and provisioning profiles centrally. It downloads certificates and profiles from a GitHub repository or Google Cloud Storage bucket. For safety, centrally stored files are encrypted.
 
+**TL;DR;**
+
+- Please do not use Match with `--readonly false` unless you really need that and you know what you are doing
+- Please don't use Xcode code signning capabilities and don't change automatic code signning in the project settings to make it download or generate code signing documents, always use Match for that
+- If you see the error when running Match make sure you specify correct parameters. Consult with `fastlane/Matchfile` and `fastlane/Appfile` to inspect defaults
+
 ## Installing Certificates and Provisioning Profiles Locally
 
 ### Day to day development
 
-For day to day development work it usually suffices to navigate to the project directory in a terminal shell and run the command
+First you need to add a developer account to your Xcode settings in `Preferences -> Accounts` menu. For that you should use a shared account (credentials are stored in the 1Password team vault). _Do not download or try to update any code signing documents via "Download Manual Profiles" or "Manage Certificates" buttons to avoid more unneeded certificates on the portal._
+
+To install development sertificates locally run
 
 ```shell
- $ fastlane match development --readonly
+ $ bundle exec fastlane match development --team_id <enterprise team id>
+```
+
+This will install latest development certificates for all the app targets listed in the `fastlane/Matchfile` in `app_identifier` function (not scoped to particular lane with `for_lane` function). 
+Match checks if the certificates and profiles are valid and will prompt you if there is an issue. By default it is configured to run in readonly mode so it will not regenerate any certificates or profiles if they are not valid.
+
+If you need to build the app locally and run it on a device that is not yet added to the development portal you need to follow these steps:
+
+- register device manully on the poratal. Make sure you do that in the correct team, it should be added to the enterprise team, not to the AppStore team
```

Comment:
That can be also done via fastlane. Command probably is the following:
```
fastlane run register_device name:<your_device_name> udid:<the_udid> team_id:<enterprise_team_id>
```

Reply:
we should probably add a lane for that (and then it can be done through slack and will be committed via shared account, so won't be a problem with access rights)

Reply:
Created a task to track that https://babylonpartners.atlassian.net/browse/CNSMR-1473